lib/MusicBrainz/Server/Edit/Utils.pm

```diff
@@ -268,14 +268,14 @@ sub changed_display_data
 }
 
 our @STATUS_MAP = (
-    [ $STATUS_OPEN         => N_l('Open') ],
-    [ $STATUS_APPLIED      => N_l('Applied') ],
-    [ $STATUS_FAILEDVOTE   => N_l('Failed vote') ],
-    [ $STATUS_FAILEDDEP    => N_l('Failed dependency') ],
-    [ $STATUS_ERROR        => N_l('Error') ],
-    [ $STATUS_FAILEDPREREQ => N_l('Failed prerequisite') ],
-    [ $STATUS_NOVOTES      => N_l('No votes') ],
-    [ $STATUS_DELETED      => N_lp('Cancelled', 'edit') ],
+    [ $STATUS_OPEN         => N_lp('Open', 'adjective, edit status') ],
```

Comment:
I'm not sure adjective is needed if edit status is already specified since they're all the same and a set? But I don't mind much either way, just thinking about consistency.

Reply:
I’m not sure it will be as clear for translators out of the code as it is for you who knows the code. The consistency is to add a disambiguation if needed, there doesn’t seem to be any ambiguity about the grammatical form for other edit statuses.